lib/capistrano/tasks/monit.cap

```diff
@@ -25,22 +25,25 @@ namespace :sidekiq do
 
     desc 'Config Sidekiq monit-service'
     task :config do
-      on roles(fetch(:sidekiq_role)) do |role|
-        @role = role
-        upload_sidekiq_template 'sidekiq_monit', "#{fetch(:tmp_dir)}/monit.conf", @role
+      Array(fetch(:sidekiq_role)).each do |role_name|
+        on roles(role_name) do |role|
+          @role = role
+          @role_name = role_name
```

Comment:
@okoriko san
I just want to question,
Are the instance variables `@role` & `@role_name` needed?